tests/x11/start_wayland_plasma5.pm

```diff
@@ -21,7 +21,7 @@ sub run {
     # Make sure everything necessary is installed
     ensure_installed "plasma5-session-wayland";
 
-    # Workaround: use softpipe as llvmpipe crashes all the time (fdo#96953)
+    # Workaround (part 1): use softpipe as llvmpipe crashes all the time (fdo#96953)
     x11_start_program 'mkdir -p ~/.config/plasma-workspace/env';
```

Comment:
Could you please add soft-failure with ticket reference here, so we don't forget about it?